deploy/sample-backup-configs/sample_config_runtime.yaml

```diff
@@ -20,11 +25,16 @@ target_config:
     net_bandwidth: 200
     profile_interval: -1
 
-    # This references a section from config_build_recipes.yaml
+    # This references a section from config_hwdb.yaml for fpga-accelerated simulation
+    # or from config_build_recipes.yaml for metasimulation
     # In homogeneous configurations, use this to set the hardware config deployed
     # for all simulators
     default_hw_config: firesim_rocket_quadcore_nic_l2_llc4mb_ddr3
 
+    # Advanced: Specify any extra plusargs you would like to provide when booting
+    # the simulator.
```

Comment:
An example or some description of the expected structure here would be good.  I'm assuming this is just a list of strings but it could just as well be a map.

Reply:
See #1077

Reply:
fixed